wordpress-mobile/WordPress-Android
# Issue/5296 image settings

Fixes #5296.

To test:
1. Open a post with an image
2. Tap on the image and open image settings
3. Change some settings & save
4. Tap on the image again
5. Verify the previous values are present
